acceptance/single_key_test.go

```diff
@@ -124,12 +124,24 @@ func TestSingleKey(t *testing.T) {
 
 	// Verify that none of the workers encountered an error.
 	var results []result
-	for i := 0; i < *numNodes; i++ {
-		r := <-resultCh
-		if r.err != nil {
-			t.Fatalf("%d: %v", i, r.err)
+	for len(results) < *numNodes {
+		delay := 1 * time.Second
+		if len(results) > 0 {
+			delay = 1 * time.Hour
```

Comment:
is this the same rationale as above? log progress every second until the first worker succeeds and then "stop" logging progress until the test completes?